mlrun/db/httpdb.py

```diff
@@ -3267,6 +3268,24 @@ def list_api_gateways(self, project=None):
         resp = self.api_call("GET", endpoint_path, error)
         return resp.json()
 
+    def create_api_gateway(
+        self,
+        project: str,
+        name: str,
+        api_gateway: mlrun.common.schemas.APIGateway,
+    ) -> bool:
+        """
+        Creates an API Gateway.
+        :param api_gateway (mlrun.runtimes.api_gateway.APIGateway): API Gateway entity.
+        :param auth (Tuple[str, str]): Pair of (username, password) if authentication is required.
+
+        @return: True if the API Gateway was created successfully, False otherwise
+        """
+        endpoint_path = f"projects/{project}/nuclio/api-gateways/{name}"
+        error = "create api gateways"
+        response = self.api_call("POST", endpoint_path, error, json=api_gateway.dict())
+        return response.ok if response else False
```

Comment:
when `create_api_gateway` will return the object, return it here as well

Reply:
@TomerShor which object?

Reply:
resolved as discussed to implement in the next PR